ecommerce-store-website/src/base/utils/interact.js

```diff
@@ -32,6 +32,12 @@ export const safeMint = async ({
 
 		window.negotiator.ui.dismissLoader()
 
+		// Automatically refresh tokens after 30 seconds
```

Comment:
Only a small thing. Can we remove the comment here. Or change it to 20 seconds

Reply:
@nicktaras good catch, done. I was tossing up whether 20 seconds would be enough for discovery providers to index. It should be in most cases.